tests/README.md

```diff
@@ -1,26 +1,72 @@
-# Mordred test environment
+# GrimoireLab test environment
 
 The goal of this directory is to include the files needed to
 use mordred to test all data sources supported in GrimoireLab.
+The files involved are:
 
-* setup.cfg: Main config file for mordred with all data sources and credentials
-* projects.json: Repositories used in data sources
-* requirements.cfg: sample file with the versions of GrimoireLab stack to be used
-* orgs_file: organizations file to be loaded in Sorting Hat for affiliations
-* cache-test.tgz: perceval cache for all data sources, mbox data and irc data.
-* docker-compose.yml: minimum docker compose file to launch mordred
-* docker-compose-local.yml: example of additional local docker compose config file
-* stage: Script for testing for the mordred docker image
+* Configuration for GrimoireLab:
+** setup.cfg: Main config file for mordred with all data sources and credentials
+** projects.json: Repositories used in data sources
+** requirements.cfg: sample file with the versions of GrimoireLab stack to be used
+** orgs_file: organizations file to be loaded in Sorting Hat for affiliations
 
-So you just need to execute:
+* Configuration for the GrimoireLab docker image:
+** stage: Script for testing for the mordred docker image
```

Comment:
stage: Script started after the docker image creation. It includes all the testing stuff.